internal/formatters/fmt.go

```diff
@@ -37,6 +37,7 @@ var (
 	undefined = models.Undefined
 	pending   = models.Pending
 	ambiguous = models.Ambiguous
+	retry     = models.Retry
```

Comment:
Whilst this is godog, and you are allowed to do your own thing, if your ultimate goal is cucumber conformance, this would need to go.

We have a concept called "flaky" which is something that passed *after* retries, maybe this can be renamed or adapted I don't know